data/dineshlalam15.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Dinesh Kumar Lalam (test)"
+  "displayStatsPublic": true,
```

Comment:
Since we no longer need the `displayStatsPublic` key, please remove this from your `json`. 
```suggestion
```